DPG/Tasks/AOTTrack/qaMatchEff.cxx

```diff
@@ -532,28 +674,61 @@ struct qaMatchEff {
       }
       //
       // all tracks, no conditions
+      if (track.hasITS() && isTrackSelectedITSCuts(track)) {
+        if constexpr (IS_MC) {
+          if (track.signed1Pt() > 0)
```

Comment:
Here you can completely avoid the two `if`, because the data model provides already a method to obtain `q/pt`. Here (and everywhere below as well) you shall fill the histogram using `track.signed1Pt()` instead of `+-1./trackPt` (see for example here: https://github.com/AliceO2Group/O2Physics/blob/master/DPG/Tasks/AOTTrack/qaEventTrack.cxx#L1121)